internal/pkg/model/path.go

```diff
@@ -5,9 +5,11 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/deepcopy"
 )
 
+type Path string
+
 type AbsPath struct {
-	RelativePath  string `json:"path" validate:"required"`
-	parentPath    string
+	RelativePath  Path `json:"path" validate:"required"`
+	parentPath    Path
```

Comment:
A tu je klucova zmena, ... pri prechadzani `state` do hlbky potrebujem odlisit `Path` od bezneho stringu.
Kedze v `Path` nenahradzujem IDs, tak sa tu typ zmenil.
To znamena, ze v ostatnom kode je potrebne pouzit `Get/Set`, ktory prevadza `string <-> Path`.